web/satellite/src/components/accessGrants/AccessGrants.vue

```diff
@@ -451,6 +455,14 @@ export default class AccessGrants extends Vue {
     public toggleAccessModal(): void {
         this.showAccessModal = !this.showAccessModal;
     }
+
+    /**
+     * Sends "trackPageVisit" event to segment and opens link.
+     */ 
+    public trackPageVisit(link: string): void {
+        this.analytics.pageVisit(link);
+        window.open(link)
+    }
```

Comment:
I want @VitaliiShpital 's opinion on opening the link like this before I approve